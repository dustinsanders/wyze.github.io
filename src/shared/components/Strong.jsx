// @flow

import type { Style, VNode } from '../../types'
import { css } from 'glamor'
import preact from 'preact'

type Props = {
  text: string,
}

const strong: Style = css({
  fontWeight: 400,
})

const Strong = ({ text }: Props): VNode => (
  <strong {...strong}>{text}</strong>
)

export default Strong
