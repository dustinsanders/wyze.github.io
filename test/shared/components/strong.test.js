// @flow

import Strong from '../../../src/shared/components/Strong'
import snapshot from '../../helpers/snapshot'

const text = 'Hello'

describe('<Strong />', () => {
  it('renders', snapshot.bind(null, Strong, { text }))
})
