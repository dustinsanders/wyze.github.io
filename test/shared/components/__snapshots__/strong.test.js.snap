// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Strong /> renders 1`] = `"<strong data-css-ucwin8>Hello</strong>"`;
